draft-irtf-cfrg-opaque.md

```diff
@@ -514,12 +525,11 @@ Once complete, the registration process proceeds as follows:
                                    response
                               <-----------------
 
- record = FinalizeRequest(pwdU, skU, blind, request, response)
+ envU, export_key = FinalizeRequest(pwdU, creds, blind, response)
 
-                                    record
+                                     envU
```

Comment:
I think this should be an instance of the `RegistrationUpload` structure, not `envU`

Reply:
I intentionally reverted this so that the wrapper could construct `RegistrationUpload`, but on reflection that's probably not best. I'll revert this bit.